.gitignore

```diff
@@ -41,6 +41,10 @@ build/Release
 node_modules/
 jspm_packages/
 
+# Build
+dist/
```

Comment:
Can you please revert the changes to this `.gitignore`. The `uplift.yml` configuration file has been updated to ensure the `dist/index.js` file is checked in and committed to any future release.

Reply:
Done